TRUSTED_TESTERS.md

```diff
@@ -436,6 +468,21 @@ You can remove `shorebird` from your path by removing it from your `.bashrc` or
 
 This section contains past updates we've sent to current Trusted Testers.
 
+## Announcement for 0.0.8
+
+We've just released Shorebird CLI v0.0.8 🎉
+
+If you're already using Shorebird there should be no change for you in
+this update.  We're just making it easier for us to add new trusted testers.
+
+What's new:
+* Added `shorebird account create` and `shorebird account subscribe` to
+  automate our onboarding process for new trusted testers.
+* Improved the way we proxy Flutter artifacts (via download.shorebird.dev) to
```

Comment:
We also updated shorebird to use a specific revision of Flutter as part of this release. Not sure if we want to mention that.

Reply:
Also added "partial" build number support 😛

Reply:
Added!